multinode_integration_tests/src/main.rs

```diff
@@ -411,22 +407,22 @@ struct PortSpec {
 }
 
 impl FromStr for PortSpec {
-    type Err = String;
+    type Err = &'static str;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         if s.len() < 2 {
-            return Err("Syntax: <U|T><port>".to_string());
+            return Err("Syntax: <U|T><port>");
```

Comment:
Given you have deployed the change where a repetitively used msg was converted into a manifest constant (see ln. 351) it would be nice, in the sake of consistency, to have it here too.

Reply:
Okay.